### PR TITLE
Some (possibly opinionated) style and layout updates

### PR DESCRIPTION
**Pull request by enclave-alistair**
A few changes here while noodling around in the docs:

### Updated Menu Layout

Docs are in one page, knowledge base and tutorials are separate:

image

Landing page is now `index.md`, so we don't always redirect to `/welcome`.

### Bullet Point Styling

Fixed bullet point styling so indentation and subsequent paragraphs align properly with the bullet, and don't use a load of calc statements to determine positioning. Change bullet colour to be primary (lighter colour felt a bit off).

Only requirement of the content is that bullets are separated by line breaks, so they become actual stylable elements, rather than just text nodes.

Before on the left, after on the right.

image

### Styled Tags

Any text surrounded with `==` is now considered a tag (e.g. `==org.any==`), and is styled as per the portal. This helps with comprehension I think between docs and portal.

Before left, after right.

image

### CODE REVIEW

**Comment by enclave-marc-barry** [comment]
Really like this; but 

1. Suggest you revert the latest commit we just made
2. Bullet points aren't working correctly everywhere 
image

**Comment by enclave-marc-barry** [comment]
Also preferred the lighter bullet colour, any reason why you darkened it?

File: docs/api/index.md [hunk 1]
```diff
@@ -31,7 +31,8 @@ are 63-character alpha-numeric strings, and are secrets that should be protected
 
 ![Location of Personal Access Tokens in your account page](/images/api/pat-location.png)
 
-> Created tokens have access to all of the Enclave organisations that your account has access to in the portal.
+> Created tokens currently have access to all of the Enclave organisations that your account has access to in the portal. In future we will allow tokens to be restricted
+> to a specific organisation.
```
Comment (enclave-marc-barry) [inline docs/api/index.md:35]: Invite readers to let us know that they want this in one of the `community-support/` channels?

File: docs/getting-started/troubleshooting.md [hunk 2]
```diff
@@ -5,22 +5,32 @@ If you find Enclave isn't working as expected, here's a simple set of troublesho
...
 7. Check network traffic isn't forced through a SOCKS proxy, which is currently unsupported
+
 8. Check local anti-virus software is not interfering with Enclave
 
 ## Peer-to-peer traffic not flowing
 
-1. Check the Enclave network adapter is online using `ipconfig /all`. Look for `Ethernet adapter Universe` in the list and check that `Media State` is not shown as `Media disconnected`. If it is, try running `enclave restart` and consider sharing local Enclave logs with [support@enclave.io](mailto:support@enclave. io) to help us investigate the problem.
+1. Check the Enclave network adapter is online using `ipconfig /all`. 
+
+    Look for `Ethernet adapter Universe` in the list and check that `Media State` is not shown as `Media disconnected`. If it is, try running `enclave restart` and consider sharing local Enclave logs with [support@enclave.io](mailto:support@enclave. io) to help us investigate the problem.
```
Comment (enclave-marc-barry) [inline docs/getting-started/troubleshooting.md:33]: I will add a note to add a documentation page about how to collect logs

Comment (enclave-marc-barry) [reply docs/getting-started/troubleshooting.md:33]: Done @ https://github.com/enclave-networks/product-management/issues/209

File: docs/index_old.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Enclave Documentation
```
Comment (enclave-marc-barry) [inline docs/index_old.md:1]: Can remove this file, we don't need it right now

File: mkdocs.yml [hunk 4]
```diff
@@ -97,21 +97,19 @@ plugins:
...
-      - Troubleshooting: getting-started/troubleshooting.md
-      
-  - Documentation:
-
+    - Getting Started:
+      - Installation: getting-started/installation.md
+      - Attach tags: getting-started/attach-tags.md
+      - Define policy: getting-started/define-policy.md
+      - Next steps: getting-started/next-steps.md
+      - Problems?:
```
Comment (enclave-marc-barry) [inline mkdocs.yml:110]: I couldn't find this. Came here expecting to comment that troubleshooting had been lost from the menu.

Can we make Troubleshooting a top-level page that jumps out in the nav pls

File: mkdocs.yml [hunk 5]
```diff
@@ -59,7 +59,8 @@ markdown_extensions:
 #      linenums: true
 #      linenums_style: pymdownx.inline
   - pymdownx.tabbed:
-      alternate_style: true 
+       alternate_style: true
```
Comment (enclave-marc-barry) [inline mkdocs.yml:62]: is this line over-indented now?

**Comment by enclave-alistair** [comment]
Changed the bullet colour out of instinct I suppose, the light blue just felt a bit odd as a list-item colour.  I don't mind changing it back, but I also wonder if we should check with Dan as a tie-break?

I've fixed the bullets that weren't working, can you provided an updated screenshot were the numbers aren't centered? The developer tools seem to indicate they are.

File: mkdocs.yml [hunk 6]
```diff
@@ -76,7 +77,6 @@ plugins:
       lang: en
   - redirects:
       redirect_maps:
-        index.md: welcome.md
```
Comment (enclave-marc-barry) [inline mkdocs.yml:79]: This leaves the current default document (`/welcome`) as a 404, no good. `/welcome` needs to redirect to `/` now

Comment (enclave-alistair) [reply mkdocs.yml:79]: Do we link to /welcome from somewhere? Or you just mean for legacy search links?

Comment (enclave-alistair) [reply mkdocs.yml:79]: Regardless, have added the redirect.

**Comment by enclave-marc-barry** [comment]
Please keep the lighter bullet colour for now. Dan will come to the docs in due course and do some wider visual updates as brand matures

re; alignment - those bullet point CSS changes seem to be creating extra work for very little gain, suggest just reverting for now back to what worked and moving on

image

image

**Comment by enclave-alistair** [comment]
Accepted on colour, I'll change them back. The main reason I changed the bullet layout previously is because they weren't centrally aligned with the first line of the text, the bottom of the item shared the same baseline as the text, which made my eyes hurt.

image

image

I'll see what I can do about adjusting the vertical position of the numbers.

**Comment by enclave-marc-barry** [comment]
> they weren't centrally aligned with the first line of the text

Damn, sorry - I assumed I'd have done a better job on that

**Comment by enclave-marc-barry** [comment]
looking much better now, but bullet points on pages with no TOC seem a little messed up still

**Comment by enclave-alistair** [comment]
I've tweaked all the comments above. The number central alignment is tricky due to kerning of the font mixing with the "center align" for different numbers. I think what we've got now is enough, but the 6 and 1 do 'look' slightly different in situ.  Take a look at it anyway, I'd hope it's sufficient.

To draw attention to the troubleshooting stuff, I've defined a new Support sub-menu (although I'm a little concerned it may be too high up in the list?). See what you think.

**Comment by enclave-marc-barry** [comment]
Look out, streams crossed. Fair point on the menu ordering

**Comment by enclave-alistair** [comment]
OK, have moved support to after setup, feels like a more natural place.

**Comment by enclave-marc-barry** [comment]
Nice, menu ordering looks better - although the bullet points still look out of whack at the bottom of the `/` page (they look fine on all of the other pages without TOC however)?

**Comment by enclave-alistair** [comment]
OK, updated CSS for bullets where no `<p>` tag is added. Believe that should fix all remaining bullet points. Have I mentioned I hate CSS?

**Review by enclave-marc-barry — APPROVED** [review]
Looks great (I added some minor additional commits btw)

**Pull request merged**